call_handlers/debugtools_commands.lua

```diff
@@ -618,4 +618,22 @@ function Commands:print_ai_stack_command(session, response, entity)
    response:resolve({'reported traceback for entity '..tostring(entity)})
 end
 
+function Commands:add_conversation_subject_command(session, response, entity, subject, polarity)
+   local conversation_component = entity:add_component('stonehearth:conversation')
+   local subject = conversation_component:add_conversation_subject(subject, polarity)
+   response:resolve(subject)
+end
+
+function Commands:get_conversation_subject_command(session, response, entity, subject)
+   local conversation_component = entity:add_component('stonehearth:conversation')
```

Comment:
yup, the conversation component is added to base humans from the json file, so you shouldn't have to add it for testing unless you're testing on goblins or something else.